cmd/internal/cli/actions.go

```diff
@@ -217,6 +184,33 @@ func replaceURIWithImage(cmd *cobra.Command, args []string) {
 	return
 }
 
+// TODO: Let's stick this in another file so that that CLI is just CLI
+func vmStarter(cmd *cobra.Command, image string, args []string) bool {
+	// check if --vm-ram or --vm-cpu changed from default value
```

Comment:
I think all of this code which is simply responsible for setting the value of `VM` should just live in it's own function. We should just be able to set the value of the flag. Then, back outside the function we'll be able to just use `is VM {...}`

Reply:
Got it, done